container-images/tcib/base/tobiko/tobiko.yaml

```diff
@@ -9,7 +9,7 @@ tcib_actions:
 - run: mkdir -p /var/lib/tempest/external_files
 - run: chown -R tobiko.tobiko /var/lib/tobiko
 - run: python3 -m pip install --upgrade pip
-- run: python3 -m pip install 'tox>=4.13'
+- run: python3 -m pip install 'tox==4.13'
```

Comment:
We are still seeing the issue with tox==4.13
```
[root@2c62695b7424 /]# python3 -m pip install 'tox==4.13'
Collecting tox==4.13
  Downloading tox-4.13.0-py3-none-any.whl.metadata (5.0 kB)
Collecting cachetools>=5.3.2 (from tox==4.13)
  Using cached cachetools-5.3.3-py3-none-any.whl.metadata (5.3 kB)
Collecting chardet>=5.2 (from tox==4.13)
  Using cached chardet-5.2.0-py3-none-any.whl.metadata (3.4 kB)
Requirement already satisfied: colorama>=0.4.6 in /usr/local/lib/python3.9/site-packages (from tox==4.13) (0.4.6)
Requirement already satisfied: filelock>=3.13.1 in /usr/local/lib/python3.9/site-packages (from tox==4.13) (3.13.1)
Requirement already satisfied: packaging>=23.2 in /usr/local/lib/python3.9/site-packages (from tox==4.13) (24.0)
Requirement already satisfied: platformdirs>=4.1 in /usr/local/lib/python3.9/site-packages (from tox==4.13) (4.2.0)
Requirement already satisfied: pluggy>=1.3 in /usr/local/lib/python3.9/site-packages (from tox==4.13) (1.4.0)
Collecting pyproject-api>=1.6.1 (from tox==4.13)
  Using cached pyproject_api-1.6.1-py3-none-any.whl.metadata (2.8 kB)
Requirement already satisfied: tomli>=2.0.1 in /usr/local/lib/python3.9/site-packages (from tox==4.13) (2.0.1)
Collecting virtualenv>=20.25 (from tox==4.13)
  Using cached virtualenv-20.25.1-py3-none-any.whl.metadata (4.4 kB)
Requirement already satisfied: distlib<1,>=0.3.7 in /usr/local/lib/python3.9/site-packages (from virtualenv>=20.25->tox==4.13) (0.3.8)
Downloading tox-4.13.0-py3-none-any.whl (155 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 155.0/155.0 kB 485.9 kB/s eta 0:00:00
Using cached cachetools-5.3.3-py3-none-any.whl (9.3 kB)
Using cached chardet-5.2.0-py3-none-any.whl (199 kB)
Using cached pyproject_api-1.6.1-py3-none-any.whl (12 kB)
Using cached virtualenv-20.25.1-py3-none-any.whl (3.8 MB)
Installing collected packages: virtualenv, pyproject-api, chardet, cachetools, tox
  Attempting uninstall: chardet
    Found existing installation: chardet 4.0.0
ERROR: Cannot uninstall chardet 4.0.0, RECORD file not found. Hint: The package was installed by rpm.

```

Reply:
If we remove and chardet and try installing above stuff it works
```
[root@2c62695b7424 /]# rpm -qa | grep chardet
python3-chardet-4.0.0-5.el9.noarch
[root@2c62695b7424 /]# dnf remove python3-chardet
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered with an entitlement server. You can use subscription-manager to register.

Dependencies resolved.
==================================================================================================================================================
 Package                                              Architecture              Version                          Repository                  Size
==================================================================================================================================================
Removing:
 python3-chardet                                      noarch                    4.0.0-5.el9                      @System                    1.4 M
Removing dependent packages:
 python3-cloud-what                                   x86_64                    1.29.40-1.el9                    @System                     76 k
 python3-requests                                     noarch                    2.25.1-8.el9                     @System                    392 k
 python3-subscription-manager-rhsm                    x86_64                    1.29.40-1.el9                    @System                    451 k
 subscription-manager                                 x86_64                    1.29.40-1.el9                    @System                    3.3 M

Transaction Summary
==================================================================================================================================================
Remove  5 Packages

Freed space: 5.6 M
Is this ok [y/N]: y
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                          1/1 
  Running scriptlet: subscription-manager-1.29.40-1.el9.x86_64                                                                                1/5 
  Erasing          : subscription-manager-1.29.40-1.el9.x86_64                                                                                1/5 
  Running scriptlet: subscription-manager-1.29.40-1.el9.x86_64                                                                                1/5 
  Erasing          : python3-subscription-manager-rhsm-1.29.40-1.el9.x86_64                                                                   2/5 
  Erasing          : python3-cloud-what-1.29.40-1.el9.x86_64                                                                                  3/5 
warning: file /var/cache/cloud-what: remove failed: No such file or directory

  Erasing          : python3-requests-2.25.1-8.el9.noarch                                                                                     4/5 
  Erasing          : python3-chardet-4.0.0-5.el9.noarch                                                                                       5/5 
  Verifying        : python3-chardet-4.0.0-5.el9.noarch                                                                                       1/5 
  Verifying        : python3-cloud-what-1.29.40-1.el9.x86_64                                                                                  2/5 
  Verifying        : python3-requests-2.25.1-8.el9.noarch                                                                                     3/5 
  Verifying        : python3-subscription-manager-rhsm-1.29.40-1.el9.x86_64                                                                   4/5 
  Verifying        : subscription-manager-1.29.40-1.el9.x86_64                                                                                5/5 

Removed:
  python3-chardet-4.0.0-5.el9.noarch                        python3-cloud-what-1.29.40-1.el9.x86_64      python3-requests-2.25.1-8.el9.noarch   
  python3-subscription-manager-rhsm-1.29.40-1.el9.x86_64    subscription-manager-1.29.40-1.el9.x86_64   

Complete!
[root@2c62695b7424 /]# python3 -m pip install 'tox>=4.13'
Collecting tox>=4.13
  Using cached tox-4.14.1-py3-none-any.whl.metadata (5.0 kB)
Collecting cachetools>=5.3.2 (from tox>=4.13)
  Using cached cachetools-5.3.3-py3-none-any.whl.metadata (5.3 kB)
Collecting chardet>=5.2 (from tox>=4.13)
  Using cached chardet-5.2.0-py3-none-any.whl.metadata (3.4 kB)
Requirement already satisfied: colorama>=0.4.6 in /usr/local/lib/python3.9/site-packages (from tox>=4.13) (0.4.6)
Requirement already satisfied: filelock>=3.13.1 in /usr/local/lib/python3.9/site-packages (from tox>=4.13) (3.13.1)
Requirement already satisfied: packaging>=23.2 in /usr/local/lib/python3.9/site-packages (from tox>=4.13) (24.0)
Requirement already satisfied: platformdirs>=4.1 in /usr/local/lib/python3.9/site-packages (from tox>=4.13) (4.2.0)
Requirement already satisfied: pluggy>=1.3 in /usr/local/lib/python3.9/site-packages (from tox>=4.13) (1.4.0)
Requirement already satisfied: pyproject-api>=1.6.1 in /usr/local/lib/python3.9/site-packages (from tox>=4.13) (1.6.1)
Requirement already satisfied: tomli>=2.0.1 in /usr/local/lib/python3.9/site-packages (from tox>=4.13) (2.0.1)
Requirement already satisfied: virtualenv>=20.25 in /usr/local/lib/python3.9/site-packages (from tox>=4.13) (20.25.1)
Requirement already satisfied: distlib<1,>=0.3.7 in /usr/local/lib/python3.9/site-packages (from virtualenv>=20.25->tox>=4.13) (0.3.8)
Using cached tox-4.14.1-py3-none-any.whl (155 kB)
Using cached cachetools-5.3.3-py3-none-any.whl (9.3 kB)
Using cached chardet-5.2.0-py3-none-any.whl (199 kB)
Installing collected packages: chardet, cachetools, tox
Successfully installed cachetools-5.3.3 chardet-5.2.0 tox-4.14.1
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
[root@2c62695b7424 /]# 
```

Reply:
If we want to remove chardet for now. We can do something like this. I think it will not affect anything
```
if [ '{{ tcib_distro }}' == 'rhel' ];then
if [ -n "$(rpm -qa redhat-release)" ];then dnf -y remove python3-chardet; fi ; fi    
```